ibis/expr/analysis.py

```diff
@@ -867,3 +875,11 @@ def _rewrite_filter_value_list(op, **kwargs):
         return op
 
     return op.__class__(*visited)
+
+
+def find_memtables(op):
```

Comment:
The analysis functions encapsulate the traversal method itself. to remain consistent could we have a `find_memtables()` with returns with an iterator of in memory tables?

Reply:
Done.

Reply:
I mean something like 

```py
def find_memtables(node)
    def finder(node):
        return g.proceed, op if isinstance(op, ops.InMemoryTable) else None

    return tuple(g.traverse(finder, filter=ops.Node))
```

Reply:
hm, I swear I implemented this.

Reply:
Done ... again 😂